src/tim/vx/ops/deconv.cc

```diff
@@ -32,33 +33,38 @@ namespace tim {
 namespace vx {
 namespace ops {
 
-DeConv2d::DeConv2d(Graph* graph, int32_t weights, PadType pad_type,
+DeConv2d::DeConv2d(Graph* graph, int32_t oc_count, PadType pad_type,
     const std::array<uint32_t, 2>& ksize,
     const std::array<uint32_t, 2>& stride,
     const std::array<uint32_t, 2>& output_padding)
-  : DeConv2d(graph, weights, pad_type, ksize, stride, output_padding,
+  : DeConv2d(graph, oc_count, pad_type, ksize, stride, output_padding,
       {0, 0, 0, 0}) {
 }
 
-DeConv2d::DeConv2d(Graph* graph, int32_t weights, PadType pad_type,
+DeConv2d::DeConv2d(Graph* graph, int32_t oc_count, PadType pad_type,
     const std::array<uint32_t, 2>& ksize,
     const std::array<uint32_t, 2>& stride,
     const std::array<uint32_t, 2>& output_padding,
-    const std::array<uint32_t, 4>& pad)
+    const std::array<uint32_t, 4>& pad,
+    const uint32_t group)
   : Operation(graph, VSI_NN_OP_DECONVOLUTION),
-    weights_(weights),
+    oc_count_(oc_count),
     pad_type_(pad_type),
     ksize_(ksize),
     stride_(stride),
     output_padding_(output_padding),
-    pad_(pad) {
+    pad_(pad),
+    group_(group) {
+
+  // TODO(Sven): only support depthwise usage
+  assert(group != 1 && group == oc_count);
```

Comment:
compiler warning for group (int32) == oc_count (uint32).

Reply:
Fixed by change 1 to 1U, and also refine the condition.